packages/yoga/src/Dialog/web/Dialog.jsx

```diff
@@ -38,7 +38,7 @@ const Overlay = styled.div`
     },
   }) => `
   display: flex;
-  z-index: 3;
+  z-index: 9;
```

Comment:
This may break other implementation which were relying on `3`.
Why not setting a new `zIndex` prop on the component and default it to `3`? Then in the style:

```suggestion
  z-index: ${props => props.zIndex};
```

Reply:
Like this? https://github.com/gympass/yoga/pull/607/commits/ce97f9202e5066d1d9349010f7ec2382153636ae